style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.box1 {
+  color: blue;
+  font-size: 25px;
+  display: block;
+  position: fixed;
+  padding: 10px;
+}
+
+.background-image {
+  padding-top: 70px;
+}
+
+.box2 {
+  color: blue;
+  display: block;
+  position: fixed;
+  right: 35px;
+  top: 30px;
+}
```

Comment:
- [ ] You should use the flexbox to position your content.  Flexbox is a great features when it comes to positioning element and it will save you the stress of responsiveness. I recommend you to remove the `Position `property and try to use flexbox, you will like how it works.  For more [information ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)